packages/frontend/src/components/common/DropDown.js

```diff
@@ -41,6 +41,8 @@ const Container = styled.div`
         }
 
         .dropdown-title {
+            overflow: hidden;
```

Comment:
Since we have the `shortenAccountId` utility function for implicit accounts, can we modify the `DropDown` component and utilize the `shortenAccountId` function so that we get the ellipsis (`xxx...xxx`) in the middle of implicit accounts?

Reply:
Only for the selected ID or also in the drop-down list?

Reply:
It'd be great to aim for consistency by having the correct ellipsis treatment (`xxx...xxx`) everywhere we show an implicit address, unless cases where showing the full address is needed or otherwise useful.

So yes, both selected ID and each account in the dropdown 👍

Reply:
I updated screens in the description. Desktop view is the same.
By the way, it wasn't a wise decision to change a title directly in the `DropDown` component. Because this is a common component and we don't know what type of title we can receive from outside.

Reply:
Great point on `DropDown` being a generic component 👍